src/System.Drawing.Primitives/src/System/Drawing/SizeF.cs

```diff
@@ -207,12 +204,13 @@ public override bool Equals(object obj)
 
             SizeF comp = (SizeF)obj;
 
-            return (comp.Width == Width) && (comp.Height == Height);
+            return (comp.Width == Width) & (comp.Height == Height);
         }
 
         public override int GetHashCode()
         {
-            return base.GetHashCode();
```

Comment:
Nice job-- it looks like this is removing a box operation every time this is called.